.github/workflows/deploy-mfe-s3.yml

```diff
@@ -50,6 +50,12 @@ jobs:
     - name: npm Install
       run: npm install
 
+    - name: pull translation
+      if: ${{ vars.ATLAS_OPTIONS != '' }}
+      run: |
+        export PATH="$(pwd)/node_modules/.bin:$PATH"
```

Comment:
This line is a good hack, I am not an expert on atlas translations but seem you need the node executable here

Reply:
https://github.com/openedx/openedx-atlas/blob/main/README.rst#installation  take a look the npm section